src/kata6.joinNames.js

```diff
@@ -1,3 +1,53 @@
-const joinNames = (namesObj) => {};
+const joinNames = (namesObj) => {
+
+  let namesArray= namesObj.map(item=>item.name)
+  
+  const namesString=namesArray.reduce((acc,curr,index)=>{
+   // const index = namesObj.findIndex((element)=>element===curr)
+    if(index!==namesObj.length-1){
+      return `${acc}, ${curr}`
+    }else{
+      return `${acc} & ${curr}`
+    }
+    
+    });
+    return namesString;
+  };
+
+
+  // const lastName=nameArray.pop();
+
+ // const sentence =  nameArray.reduce((acc,curr)=>{
+   //   const index = namesObj.findIndex((element)=>element===curr)
+     // return const lastValue = `& ${lastName}`
+      //if (index!==nameArray.length-1){
+        // return `${acc}, ${curr}`
+     // }
+      //return sentence + lastValue;
+
+
+
+
+ //const newObj = namesObj.map((object)=>object.name).reduce((acc,curr)=>{
+ // const index = namesObj.findIndex((element)=>element===curr)
+   //const lastObj=newObj.pop()
+   //return `${acc} , ${curr} & ${lastObj}`
+  // + last object
+     
+     
+
+    
+    
+    //.reduce((acc,curr)=>{
+    // if(index!==namesObj.length-1){
+      //  return `${acc} ${curr}`
+///}else{
+      //    return `${acc} & ${curr}`
+    //   }
+    //})
+//}
+
+
+
```

Comment:
Once again, this is just formatting and removing all the extra clear lines and comments
```
    it("returns the employee's role in the company", () => {
        expect(getEmployerRole("Satti", employees)).toEqual("Developer");
        expect(getEmployerRole("Jenny", employees)).toEqual("Sales Associate");
        expect(getEmployerRole("Javid", employees)).toEqual(
            "Human Recommended Reading Assistant"
        );
    });
```